eng/common/scripts/Package-Properties.ps1

```diff
@@ -105,6 +105,28 @@ function Get-PkgProperties
     return $null
 }
 
+function Get-PrPkgProperties([string]$InputDiffJson) {
+    $packagesWithChanges = @()
+
+    $allPackageProperties = Get-AllPkgProperties -ServiceDirectory media
+    $diff = Get-Content $InputDiffJson | ConvertFrom-Json
+    $targetedFiles = $diff.ChangedFiles
+
+    foreach($pkg in $allPackageProperties)
+    {
+        foreach($file in $targetedFiles)
+        {
+            $filePath = Resolve-Path (Join-Path $RepoRoot $file)
+            $shouldInclude = $filePath -like "$($pkg.DirectoryPath)*"
```

Comment:
This should work as long as pkg.DirectoryPath is also a Resolved-Path, which I'm not sure but we can resolve it if needed.

Reply:
You're right. I'll resolve it here just to be certain.

Reply:
![Uploading image.png…]()